packages/api/endpoints/distribution.js

```diff
@@ -83,58 +84,83 @@ async function handleRedirectRequest(req, res) {
     username: getAccessTokenResponse.username
   });
 
-  return res.cookie(
-    'accessToken',
-    getAccessTokenResponse.accessToken,
-    {
-      expires: new Date(getAccessTokenResponse.expirationTime),
-      httpOnly: true,
-      secure: true
-    }
-  )
+  return res
+    .cookie(
+      'accessToken',
+      getAccessTokenResponse.accessToken,
+      {
+        expires: new Date(getAccessTokenResponse.expirationTime),
+        httpOnly: true,
+        secure: true
+      }
+    )
     .set({ Location: urljoin(distributionUrl, state) })
     .status(307)
     .send('Redirecting');
 }
 
 /**
- * Responds to a file request
+ * Ensure request is authorized through EarthdataLogin or redirect to become so.
  *
  * @param {Object} req - express request object
  * @param {Object} res - express response object
- * @returns {Promise<Object>} the promise of express response object
+ * @param {Function} next - express middleware callback function
+ * @returns {Promise<Object>} - promise of an express response object
  */
-async function handleFileRequest(req, res) {
+async function ensureAuthorizedOrRedirect(req, res, next) {
   const {
     accessTokenModel,
-    authClient,
-    s3Client
+    authClient
   } = getConfigurations();
 
-  const redirectToGetAuthorizationCode = res
-    .status(307)
-    .set({ Location: authClient.getAuthorizationUrl(req.params[0]) });
+  const redirectURLForAuthorizationCode = authClient.getAuthorizationUrl(req.params[0]);
 
   const accessToken = req.cookies.accessToken;
 
-  if (!accessToken) return redirectToGetAuthorizationCode.send('Redirecting');
+  if (!accessToken) return res.redirect(307, redirectURLForAuthorizationCode);
```

Comment:
It doesn't matter, but did you think this syntax was easier to read?

Reply:
To me, it feels a little more "expressy" calling `redirect`, rather than `.send('redirect!')`.
The original refactor with the old notation, required me to eliminate the settng of `status(307)` on https://github.com/nasa/cumulus/pull/881/files/13a9eae47b9d3ee7f867bbf9ce8046f8016863d3#diff-a839d1b4978cafe0031568b43ce88e1bL115
and then adding that status(307).send('redirect') to all of the redirects.

so instead of
```
res.redirect(307, redirectURLForAuthorizationCode);
```
They were
```
redirectToGetAuthorizationCode.status(307).send('redirecting').
```

I can but that back, because I don't have much loyalty to either since I haven't worked in express much.

Reply:
Nope, it's good like this. I was just curious